lib/Sema/SemaBounds.cpp

```diff
@@ -369,7 +445,7 @@ namespace {
                                                SourceLocation());
         }
         case BoundsExpr::Kind::InteropTypeAnnotation:
-          return CreateBoundsNone();
+          return CreateBoundsAllowedButUncomputable();
```

Comment:
This is somewhere where we used to compute `bounds(none)` and now compute `bounds(any)`, because we want to allow this, but cannot express these bounds at the moment.